### PR TITLE
[arser] Revise CMakeLists

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise CMakeLists file; use nncc_coverage option and test to
use arser lib for include folder.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #5960, from draft #5961

File: compiler/arser/CMakeLists.txt [hunk 1]
```diff
@@ -12,4 +13,4 @@ endif(NOT ENABLE_TEST)
 nnas_find_package(GTest REQUIRED)
 set(TESTS "${CMAKE_CURRENT_SOURCE_DIR}/tests/arser.test.cpp")
 GTest_AddTest(arser_test ${TESTS})
-target_include_directories(arser_test PRIVATE include)
+target_link_libraries(arser_test arser)
```
Comment (mhs4670go) [inline compiler/arser/CMakeLists.txt:16]: I'm not sure but does it have same meaning?

Comment (seanshpark) [reply compiler/arser/CMakeLists.txt:16]: Yup, `arser` has `target_include_directories(arser INTERFACE include/)` and linking `arser` will include to `include/` folder

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**